syntax = "proto3";

service HeartBeatService {
	rpc HeartBeat(HeartBeatRequest) returns (HeartBeatResponse) {}
}

message HeartBeatRequest {
	int32 storage = 1;	
	int32 pid = 2;
	string ip_address = 3;
}

message HeartBeatResponse {
	bool ack = 1;	
}
